index.html

```diff
@@ -10,6 +10,12 @@
     out in the same tree and use relative links so that they'll work offline.
    -->
 
+  <script src="https://www.w3.org/scripts/jquery/2.2/jquery.min.js" crossorigin="anonymous"></script>
```

Comment:
```suggestion
  <script src="https://www.w3.org/scripts/jquery/2.2.4/jquery.min.js" crossorigin="anonymous"></script>
```

Reply:
This change fixed the jquery loading but the datatables functions are still not firing as expected, with no errors in the console that I can see that would be applicable. I'm only getting errors for the icons from datatables, which is equally confusing.

Reply:
This shouldn't be necessary unless we expect a later dot-release of 2.2.x to break things.  <https://www.w3.org/scripts/jquery/2.2/> resolves to <https://www.w3.org/scripts/jquery/2.2.4/>, and will continue to resolve to the latest `2.2.x`.

Reply:
You'd think that, but the CORS request failed when it was pointed at `2.2` and works with `2.2.4`, even though Datatables still isn't working with either.

Reply:
https://www.w3.org/scripts/jquery/2.2/ is actually a temporary redirect to https://www.w3.org/scripts/jquery/2.2.4/.
I believe the reason it's failing when using the former URL is because of the crossorigin attribute. Can you try to remove it to see if it's working better?